### ISSUE TITLE
DB_URI parsing error and SMTP have to be an option

### ISSUE THREAD

**Issue by haydnhkim** [macropin/docker-strider#3]
I was passing a DB_URI argument like below

```
-e DB_URI=mongodb://id:password@host.mongohq.com:10047/db
```

and it parsed result was wrong.

```
strider@d03ceae1d912:/$ echo "$(echo ${DB_URI} | cut -d/ -f3 | cut -d \: -f2)"
password@host.mongohq.com

strider@d03ceae1d912:/$ echo "$(echo ${DB_URI} | cut -d/ -f3 | cut -d \: -f1)"
id
```

<br>
I don't want to receive email notifications, but I got a message like 'You must link this container with SMTP or define SMTP_HOST'.
Official Strider-CD repository wrote SMTP configure does not required.

> If you want email notifications, configure an SMTP server
> https://github.com/Strider-CD/strider#configuring

**Comment by macropin** [comment]
An SMTP service is required for basic functionality such as password reset, or activating a new user account via invitation.

**Comment by haydnhkim** [comment]
@macropin 
How can I passed a mongodb id/pw arguments?
[L43](https://github.com/macropin/docker-strider/blob/master/entry.sh#L43) and [L44](https://github.com/macropin/docker-strider/blob/master/entry.sh#L44) code can't split `DB_URI=mongodb://id:password@host.mongohq.com:10047/db` string.

This parsed result still wrong.

**Comment by macropin** [comment]
Ah ok. That should be fixed in caac2d8666d3a16fa416acae84a01e37c279c39e

Please check the latest build when it's finished building, and let me know how you go.

**Comment by haydnhkim** [comment]
Thanks! :+1:
